gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=28.3.5
+version=28.3.6
```

Comment:
You may need a major version bump here due to the interface changes.

Reply:
We decided to avoid a major version bump since TrackerClient is an internal class and external MPs should not be affected (verified through code search).